### ISSUE TITLE
Calling window when importing module

### ISSUE THREAD

**Issue by leognmotta** [infinitered/reactotron-react-js#153]
is it possible not to call window when importing reactotron-react-js, just on configure?

import Reactotron from 'reactotron-react-js' // calls window

Reactotron.configure() // only calls window here

**Comment by leomotta121** [comment]
https://github.com/gatsbyjs/gatsby/issues/19720

**Comment by leomotta121** [comment]
I actually have identified, that it is indeed calling the window, I would suggest wrapping it in a window check, something like `if(typeof window !== 'undefiened')`.

here is what I did to make it work:

```js
/* eslint-disable no-restricted-globals */
import { createClient } from 'reactotron-core-client'
import { reactotronRedux } from 'reactotron-redux'
import reactotronSaga from 'reactotron-redux-saga'

import isBrowser from '@helpers/isBrowser'

function getNavigatorProperty(name) {
  if (!name) return undefined
  if (isBrowser()) return undefined
  if (!window.navigator && typeof window.navigator !== 'object')
    return undefined
  return window.navigator[name]
}

if (process.env.NODE_ENV === 'development' && isBrowser()) {
  const REACTOTRON_ASYNC_CLIENT_ID = '@REACTOTRON/clientId'

  const client = {
    createSocket: path => new WebSocket(path), // eslint-disable-line
    host: 'localhost',
    port: 9090,
    name: 'React JS App',
    client: {
      reactotronLibraryName: 'reactotron-react-js',
      reactotronLibraryVersion: 'REACTOTRON_REACT_JS_VERSION',
      platform: 'browser',
      platformVersion: getNavigatorProperty('platform'),
      userAgent: getNavigatorProperty('userAgent'),
      screenWidth: (screen && screen.width) || undefined,
      screenHeight: (screen && screen.height) || undefined,
      screenScale: (window && window.devicePixelRatio) || 1,
      windowWidth: (window && window.innerWidth) || undefined,
      windowHeight: (window && window.innerHeight) || undefined,
    },
    getClientId: () => {
      return Promise.resolve(localStorage.getItem(REACTOTRON_ASYNC_CLIENT_ID))
    },
    setClientId: clientId => {
      localStorage.setItem(REACTOTRON_ASYNC_CLIENT_ID, clientId)
      return Promise.resolve()
    },
  }

  const Reactotron = createClient(client)

  const tron = Reactotron.configure()
    .use(reactotronRedux())
    .use(reactotronSaga())
    .connect()

  tron.clear()

  console.tron = tron
}

```

**Comment by kevinvangelder** [comment]
Resolved by #862
